torchvision/models/detection/rpn.py

```diff
@@ -298,8 +311,17 @@ def _get_top_n_idx(self, objectness, num_anchors_per_level):
         r = []
         offset = 0
```

Comment:
If we make `_get_top_n_idx` a torchscript function (moving it outside of the body of the class), would it work with ONNX? The function can be entirely made torchscript ready (only need to add the annotations).

Reply:
You are suggesting to keep the old implementation of _get_top_n_idx but as a torchscript function, correct?

The use of the native python function min() will still be an issue here, and will not be supported by onnx.
This suggested implementation replaces the native python min() with torch.min which is supported by onnx.

Reply:
Yes, it's true that it might require some further changes to the way ONNX handles things.

Here is my thinking (maybe it doesn't make sense, please let me know):

AFAIK ONNX only supports tensors, and not numbers.
During conversion from TorchIR to ONNX IR, we perform an upcast from all numbers to tensors. We also replace `prim::min` by `aten::min`, so that the same code could be used by both ONNX and TorchScript?

Does this seem feasible?